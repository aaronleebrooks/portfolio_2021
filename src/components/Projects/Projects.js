import styles from "./index.module.css";
import cx from "classnames";

const theme = {
  root: cx(
  )
}

function Projects() {
  return (
    <div className={theme.root}>
      Projects
    </div>
  );
}

export default Projects;
