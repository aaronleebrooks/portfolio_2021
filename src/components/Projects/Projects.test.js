import { render } from '@testing-library/react';
import Projects from './Projects';

describe('Projects', () => {
  test('renders', () => {
    const { container } = render(<Projects />);
    expect(container).toMatchSnapshot();
  });
})
