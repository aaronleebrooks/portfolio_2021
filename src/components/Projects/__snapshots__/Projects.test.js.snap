// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders 1`] = `
<div>
  <div
    class=""
  >
    Projects
  </div>
</div>
`;
